packages/auth/src/__tests__/buildAccessPolicy/helpers.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  findOrCreateDummyRecord,
+  getTestDatabase,
+  ModelRegistry,
+  upsertDummyRecord,
+} from '@tupaia/database';
+
+export const setUp = async () => {
```

Comment:
I could also name this file `setup.js` as you have done, but we generally use the convention of a `helpers.js` so I ended up using this.

Other conventions you may see across the codebase:

* `${testName}.fixtures.js` for test fixtures (=> test data)
* `${testName}.stubs.js` for stubs

Reply:
> I could also name this file `setup.js` as you have done, but we generally use the convention of a `helpers.js` file for similar functionality, so I ended up using this.
> 
> Other conventions you may see across the codebase:
> 
> * `${testName}.fixtures.js` for test fixtures (=> test data)
> * `${testName}.stubs.js` for stubs

Roger that. So nice you list it all! Thanks Kostas!

Reply:
Perhaps it would be good to document this kind of information... Although the problem with documentation is that it often becomes obsolete even before you finish writing it!